sdk/nestjs/tsconfig.lib.json

```diff
@@ -8,9 +8,8 @@
     "exclude": [
         "**/*.spec.ts",
         "**/*.test.ts",
-        "**/__mocks__/**/*.ts",
         "**/__tests__/**/*.ts",
         "jest.config.ts"
     ],
-    "include": ["**/*.ts"]
+    "include": ["**/*.ts", "mocks.ts"]
```

Comment:
does `**/*.ts` not already satisfy the path for `mocks.ts`?

Reply:
yep, you're right — I think when I originally added this the file was `mocks.js`